javascript/javascript3/week1/js3week1-gif.js

```diff
@@ -0,0 +1,22 @@
+let displayGifs = document.getElementById("displayGifs");
+
+function searchGifs() {
+  displayGifs.innerHTML = " ";
+  let searchWord = document.getElementById("searchWord").value;
+  let limitGifNumber = document.getElementById("noOfGifs").value;
+  if (searchWord == 0 || searchWord == " ") {
```

Comment:
You should check the length of the searchWord. Also if you can use trim here, then it will remove all space before and after text
```suggestion
  if (searchWord.trim().length === 0) {
```